Cache/Memcached.php

```diff
@@ -254,54 +238,35 @@ protected function getDataFromPayload($payload)
 		return $payload;
 	}
 
-	/**
-	 * Sets the prefix for this client
-	 *
-	 * @param string $prefix Prefix to use for this client
-	 *
-	 * @return Memcached
-	 */
-	public function setPrefix($prefix)
+	public function setPrefix(string $prefix): Memcached
 	{
 		$this->prefix = $prefix;
 
 		return $this;
 	}
 
-	/**
-	 * @return string Returns the prefix
-	 */
-	public function getPrefix()
+	public function getPrefix(): string
 	{
 		return $this->prefix;
 	}
 
-	/**
-	 * @return bool Returns whether or not $this->prefix is empty()
```

Comment:
I'm inclined to normally leave these kind of statements if they contain "useful" commentary. Arguably this just about does. Not wedded to it though.